content/reference-currency.md

```diff
@@ -0,0 +1,60 @@
+# Currency Formats
+
+The XRP Ledger has [two kinds of money](concept-money.html): XRP, and issued currencies. In the XRP Ledger, both types have high precision, although their formats are different.
+
+## String Formatting
+
+{% include 'snippets/string-number-formatting.md' %}
+
+## XRP Precision
+
+XRP has the same precision as a 64-bit unsigned integer where each unit is equivalent to 0.000001 XRP. Its properties are:
+
+* Minimum value: `0`
+* Maximum value: `100000000000` (10<sup>11</sup>) XRP
+    - `"100000000000000000"` (10<sup>17</sup>) drops of XRP
+* Precise to the nearest `0.000001` (10<sup>-6</sup>) XRP
+    - `"1"` drop of XRP
+
+## Issued Currency Precision
+
+Issued currencies in the XRP Ledger are represented with a custom format with the following precision:
+
+* Minimum nonzero absolute value: `1000000000000000e-96`
+* Maximum value: `9999999999999999e80`
+* Minimum value: `-9999999999999999e80`
+* 15 decimal digits of precision
+
+## Issued Currency Math
+[[Source]<br>](https://github.com/ripple/rippled/blob/35fa20a110e3d43ffc1e9e664fc9017b6f2747ae/src/ripple/protocol/impl/STAmount.cpp "Source")
+
+![Issued Currency Amount Format diagram](img/currency-number-format.png)
+
+Internally, `rippled` represents numbers for issued currencies in a custom number format. This format can store a wide variety of assets, including those typically measured in very small or very large denominations. Unlike typical floating-point representations of non-whole numbers, this format uses integer math for all calculations, so it always maintains 15 decimal digits of precision. Unlike "arbitrary precision" number formats, the custom format can always be stored in a fixed size of 64 bits.
+
+The internal format consists of three parts: a sign bit, a mantissa, and an exponent. The mantissa contains up to 15 significant digits (in base-10), and the exponent contains a scale (from -96 to +80). The sign bit indicates whether the amount is positive or negative. Before recording any amount, `rippled` "canonicalizes" the value so that the mantissa is within the range `1000000000000000` to `9999999999999999` (inclusive) and the exponent is in the range -96 to 80 (inclusive). (Exception: the value `0` is represented with a mantissa of `0`.) For example, the canonical representation of 1 unit of currency is `1000000000000000e-15`. The internal calculations generally use integer math so that numbers are always precise within 15 digits.
```

Comment:
_comment from ignorance_ Anyone reading this page would know what a mantissa is, right? (I had to look it up.)

Reply:
Not necessarily, unfortunately. It's something I had to look up, too. Sadly, the definitions I found were in some ways more confusing than ever.

Looking it over again, it's possible to rephrase the entire section to use "significant digits" exclusively and avoid the word "mantissa", which is probably more accessible to more readers, so I'm going to do that.